### DISCUSSION TITLE
Ditto Printing / Simultaneous Dual Extrusion bounty

### DISCUSSION THREAD

**Discussion by AskAlice** [moonglow/FlashForge_Marlin#110] [Ideas]
I'll put $50 down for this to be added

**Comment by OneMonkeyArmy** [comment]
Are you talking about 2 extruders/or 2hotends ( Bowden extruder) on the same track, that can work independently / mirror / simultaneous... if it is that .. it is called IDEX, and I just finish my prototype on a 3D20, without adding a 2nd stepper for moving the 2 hotends....  go figure it out...

**Reply by AskAlice** [reply]
i have dual extruders and i think it's just a matter of telling them to both extrude the same line

**Comment by AskAlice** [comment]
there's this which has ditto printing. basically would be merging it. problem is the code is 7 years old and thee structure of marlin has changed a lot. 

https://github.com/ShaqFoo/Marlin_Firmware_Deluxe_With_Ditto_Print-Auto_Leveling

**Comment by moonglow** [comment]
@AskAlice  HI !.... i need to read about it more ,but looks like it gives you only small area for printing by two heads

**Reply by AskAlice** [reply]
quite useful for thin things like this https://www.thingiverse.com/thing:810373

honestly, loads of functional prints fit within that size limit. Less so with pretty things
